packages/server/modules/workspaces/graph/resolvers/workspaces.ts

```diff
@@ -312,7 +331,61 @@ export = FF_WORKSPACES_MODULE_ENABLED
 
           return await getWorkspaceFactory({ db })({ workspaceId })
         },
-        invites: () => ({})
+        invites: () => ({}),
+        addDomain: async (_parent, args, context) => {
+          await authorizeResolver(
```

Comment:
No, since the auth is authorizing a resolver it doesn't belong to the service. Services should never know anything about the application context they are running in. So importing graphql types, resolvers and all of that int the service breaks that.
Plus, with the requires role, requires scope we're already doing the authz in the app layer.